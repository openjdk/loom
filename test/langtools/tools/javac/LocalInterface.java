/**
 * @test  /nodynamiccopyright/
 * @bug 8242478 8246774
 * @summary test for local interfaces
 * @compile/fail/ref=LocalInterface.out -XDrawDiagnostics --release 15 LocalInterface.java
 * @compile LocalInterface.java
 */
class LocalInterface {
    void m() {
        interface I {}
    }
}

