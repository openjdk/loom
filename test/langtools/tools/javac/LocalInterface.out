LocalInterface.java:10:9: compiler.err.intf.not.allowed.here
1 error
